core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java

```diff
@@ -68,8 +68,10 @@
  *    stored in mSyncPathStatus.
  */
 @NotThreadSafe
-public class ActiveSyncManager implements JournalEntryIterable, JournalEntryReplayable {
+public class ActiveSyncManager implements Journaled {
```

Comment:
@yuzhu should take a look at this.

Reply:
@aaudiber Thanks for the refactoring. the part related to active sync journaling LGTM.